openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.test.tsx

```diff
@@ -91,9 +102,21 @@ describe('Test CustomProperty Table Component', () => {
     (getTypeByFQN as jest.Mock).mockImplementationOnce(() =>
       Promise.resolve({ customProperties: [] })
     );
-    const { findByText } = render(<CustomPropertyTable {...mockProp} />);
-    const noDataPlaceHolder = await findByText('ErrorPlaceHolder.component');
+    await act(async () => {
+      render(<CustomPropertyTable {...mockProp} />);
+    });
+    const noDataPlaceHolder = await screen.findByText(
+      'ErrorPlaceHolder.component'
+    );
 
     expect(noDataPlaceHolder).toBeInTheDocument();
   });
+
+  it('Loader should be shown while loading the custom properties', async () => {
+    (getTypeByFQN as jest.Mock).mockResolvedValueOnce(Promise.resolve({}));
+    render(<CustomPropertyTable {...mockProp} />);
+
+    // To check if loader was rendered when the loading state was true and then removed after loading is false
+    await waitForElementToBeRemoved(() => screen.getByTestId('loader'));
```

Comment:
don't we need assertion here?

Reply:
Actually the waitForElementToBeRemoved function did the assertion that we needed, but I'll add one more assertion for what is displayed after loading.

Reply:
Thanks!